IntegrationTests/src/bkr/inttest/server/tools/test_refresh_ldap.py

```diff
@@ -25,6 +25,9 @@ def test_version(self):
         self.assertEquals(out.strip(), __version__)
 
     def test_refresh_ldap_group_membership(self):
+        # Testing two things with this test.  That jgillard is
+        # learned AND user 'nomailattr' is NOT learned since it is
+        # missing the 'mail' attribute.
         with session.begin():
```

Comment:
Both are members of the Australia ldap group.  Having both at the same time shows the ldap group
was processed successfully since jgillard is still in the list AND it proves at the same time that
NoMail was bypassed and the bogus NoMail user config causes no harm.

Reply:
For your first question on the comments,  aren't you  happy I microcoded less then I wrote 'C' code?

Reply:
About the comments, I likely mimicked what I saw in the setUpClass earlier in the method.